src/platform/packages/shared/kbn-coloring/src/shared_components/color_mapping/components/palette_selector/palette_selector.tsx

```diff
@@ -100,6 +100,7 @@ export function PaletteSelector({ palettes }: { palettes: KbnPalettes }) {
               'data-test-subj': `kbnColoring_ColorMapping_Palette-${palette.id}`,
               value: palette.id,
               title: palette.name,
+              append: getAppendedTag(palette.tag),
               palette: Array.from({ length: palette.colorCount }, (_, i) => {
                 return palette.getColor(i);
               }),
```

Comment:
The true intent was to be able to set a `tag` string on the main palette definitions to be displayed in the palette picker. I added these changes in 6ff7d3d2afde6667700b66fc2ea2759449c55bfb.

This adds the `Default` tag to the elastic palette in all pickers in Lens, TSVB and aggs-based.

https://github.com/user-attachments/assets/5bd9290a-f42a-4b6b-b3bf-db93631b1dd7

@MichaelMarcialis are these changes ok with you?

Reply:
Side note -- there are so many places in kibana that implement their own palette picker it's too much effort to apply this to all. Ideally in the next 6 months we have time to fix this and provide better alignment for palettes across kibana.

Reply:
Looks great to me. Thanks, @nickofthyme!